region/norfair/east/Cathedral Entrance.json

```diff
@@ -263,6 +347,78 @@
         "With heat protection, it should be possible to farm a bit to get out of health bomb range."
       ]
     },
+    {
+      "link": [2, 1],
+      "name": "G-Mode",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": false
+        }
+      },
+      "requires": [
+        "Morph",
+        {"or": [
+          "h_canBombThings",
+          "h_canUseSpringBall",
+          {"and": [
+            "canNeutralDamageBoost",
+            {"enemyDamage": {"enemy": "Sova", "type": "contact", "hits": 1}},
+            {"or": [
+              {"and": [
+                "canTrickyJump",
+                "canLateralMidAirMorph",
+                "canCameraManip"
+              ]},
+              "canBePatient",
+              {"ammo": {"type": "Super", "count": 1}}
+            ]}
+          ]}
+        ]},
+        {"or": [
+          "canPauseAbuse",
+          "h_HeatedGModeOpenDifferentDoor"
+        ]},
+        {"heatFrames": 0}
+      ],
+      "flashSuitChecked": true,
+      "note": [
+        "Cross the room and go through the morph tunnel.",
+        "Without any morph movement items, carefully air ball over the Sova and use it to damage boost into the tunnel or wait for the global Sova.",
+        "It is possible to kill a Sova on the door and pause abuse to grab its Energy drop on G-mode exit."
+      ]
+    },
+    {
+      "link": [2, 1],
+      "name": "G-Mode Morph",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": true
+        }
+      },
+      "requires": [
+        {"or": [
+          "canTrickyDodgeEnemies",
+          "h_canArtificialMorphPowerBomb"
+        ]},
+        {"or": [
+          "h_canArtificialMorphIBJ",
+          "h_canArtificialMorphSpringBall"
+        ]},
+        {"or": [
+          "canPauseAbuse",
+          "h_HeatedGModeOpenDifferentDoor"
+        ]},
+        {"heatFrames": 0}
+      ],
+      "flashSuitChecked": true,
+      "note": [
+        "Carefully roll under the Sm. Desgeegas or kill them with a Power Bomb (while avoiding killing the Sova).",
+        "It is possible to kill a Sova on the door and pause abuse to grab its Energy drop on G-mode exit.",
+        "With Spring Ball or a moderately good IBJ, Samus should be able to get through the morph tunnel before the global Sova, otherwise it takes just over a minute for it to return."
```

Comment:
I found it easier to use the local sova.  You have to not bring it on camera then do a diagonal horizontal bomb boost to go up both steps at once